geoh5py/objects/object_base.py

```diff
@@ -438,6 +480,25 @@ def remove_property_groups(
 
         self.workspace.update_attribute(self, "property_groups")
 
+    def get_property_group(self, name: uuid.UUID | str) -> list:
```

Comment:
I like that. We should eventually deprecate the `find_or_create_property_group` method in favor of two distinct methods...

Reply:
Find or create property is interesting in a way. 
I just split this function in find_property and create_property and merge them both in the find_or_create_property group. 
Make the code clearer.